generated/nidaqmx/nidaqmx.proto

```diff
@@ -3913,9 +3913,12 @@ enum DeviceInt32AttributeValues {
   DEVICE_INT32_CO_OUTPUT_TYPE_PULSE_TIME = 10269;
   DEVICE_INT32_CO_OUTPUT_TYPE_PULSE_FREQ = 10119;
   DEVICE_INT32_CO_OUTPUT_TYPE_PULSE_TICKS = 10268;
-  DEVICE_INT32_COUPLING1_AC = 10045;
-  DEVICE_INT32_COUPLING1_DC = 10050;
-  DEVICE_INT32_COUPLING1_GND = 10066;
+  DEVICE_INT32_COUPLING_TYPES_AC = 1;
```

Comment:
@bkeryan the field Ids are not unique in the newly added fields (`DEVICE_INT32_COUPLING_TYPES`  and `DEVICE_INT32_TRIGGER_USAGE_TYPES`). Any suggestions on how to resolve this?

Reply:
The codegen automatically specifies `allow_alias` when the enum contains duplicate values. See https://github.com/ni/grpc-device/blob/7927feb2d590821a8066b82ae5b34677f312944b/source/codegen/proto_helpers.py#L26

Is this causing a problem for your work?

Reply:
Attribute values enums will almost always have overlaps and `allow_alias`. It should be rare otherwise. `allow_alias` is not recommended in general.

Reply:
No Brad, this doesn't cause any issue. But as a general standard the field Ids should be unique in proto file, hence I wanted to confirm this.